spec/system/admin/instrument_price_policies_controller_spec.rb

```diff
@@ -358,28 +357,46 @@
       expect(page).to have_field("price_policy_#{base_price_group.id}[duration_rates_attributes][1][rate]", with: "40.00")
       expect(page).to have_field("price_policy_#{base_price_group.id}[duration_rates_attributes][2][rate]", with: "30.00")
 
-      fill_in "min_duration_2", with: "5"
-      fill_in "price_policy_#{base_price_group.id}[duration_rates_attributes][2][rate]", with: "20"
+      # Update values for Step 2
+      fill_in "min_duration_1", with: "5"
+      fill_in "price_policy_#{base_price_group.id}[duration_rates_attributes][1][rate]", with: "20"
+
+      # Remove all values for Step 3
```

Comment:
What would you write in the `it` description if you had one here?  Is this accurate?
```
# When values for Step 3 are removed, you can still save Step 1 and 2
```